sdk/computefleet/arm-computefleet/api-extractor.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "./dist/esm/index.d.ts",
+  "docModel": {
+    "enabled": true
+  },
+  "apiReport": {
+    "enabled": true,
+    "reportFolder": "./review"
+  },
+  "dtsRollup": {
+    "enabled": true,
+    "untrimmedFilePath": "",
+    "publicTrimmedFilePath": "./types/arm-computefleet.d.ts"
```

Comment:
We don't have a types folder anymore, that it should be fine to put it in the `dist` folder

Reply:
@mpodwysocki  I am a little confused about the ask here, I do see the types folder if I finish the build. And there's four different folders under dist folder, but none of them seems equivalent to the one in the types folder? Also OpenAI modular is using the same way https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/openai/openai/api-extractor.json#L9, not sure if I am missing anything ? 
<img width="484" alt="image" src="https://github.com/user-attachments/assets/e8134ce6-e274-4965-b1b6-1ffe7ccd5b17">